contributors/devel/sig-release/release.md

```diff
@@ -107,7 +107,11 @@ be consistent across artifact types.
   contact info for the team associated with any given release can be
   found [here](https://git.k8s.io/sig-release/releases/).
 - *Y days*: Refers to business days (using the location local to the release-manager M-F).
-- *feature*: see "[Is My Thing a Feature?](http://git.k8s.io/features/README.md#is-my-thing-a-feature)
+- *enhancement*: see "[Is My Thing an Enhancement?](http://git.k8s.io/enhancements/README.md#is-my-thing-an-enhancement)
+- *[Enhancement Freeze](git.k8s.io/sig-release/releases/release_phases.md#enhancements-freeze)*: Usually during Week 4 of a release cycle: the deadline by which KEPs have to be completed in order for enhancements to be part of the current release
+- *[Exception Request](git.k8s.io/sig-release/releases/release_phases.md#exceptions)*: The process of requesting an extension on the deadline for a particular Enhancement
+- *[Code Freeze](git.k8s.io/sig-release/releases/release_phases.md#code-freeze)*: The period of ~4 weeks before the final release date, during which only critical bug fixes are merged into the release.
+- *[Pruning](git.k8s.io/sig-release/releases/release_phases.md#pruning)*: The process of removing an Enhancement from a release milestone if it is not fully implemented or is otherwise considered not stable.
```

Comment:
```suggestion
- *[Pruning](https://git.k8s.io/sig-release/releases/release_phases.md#pruning)*: The process of removing an Enhancement from a release milestone if it is not fully implemented or is otherwise considered not stable.
```